packages/cdr/docker/jsreport/scripts/report-5f/content.js

```diff
@@ -0,0 +1,186 @@
+const axios = require('axios')
+
+const ES_INDEX = 'fhir-report-patient'
+const ES_USERNAME = 'elastic'
+const ES_PASSWORD = 'dev_password_only'
+
+async function beforeRender(req) {
+  const { from, to, state, district, city, facilityCode } = req.data.params
+
+  const esQuery = {
+    size: 0,
+    query: {
+      bool: {
+        filter: [
+          {
+            range: {
+              'artInitiation.dateInitiated': {
```

Comment:
This field has changed in the latest index to be `artInitiation.date`. If you refresh you environment you will see this query doesn;t work unless you correct this field.